### PR TITLE
Kea DHCP workload using Podman (jsc#DOCTEAM-922)

### PR DESCRIPTION
**Pull request by tbazant**
### Are there any relevant issues/feature requests?

* jsc#DOCTEAM-922

### CODE REVIEW

**Comment by jcronenberg** [comment]
Something that may be worth mentioning, which I need to add to the README in OBS as well, is that for DHCPv6 if firewalld is running, you may need to add a DHCPv6 rule.
```
firewall-cmd --add-service=dhcpv6 ...
```

File: tasks/kea-dhcp-running-with-podman.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+  <info>
+    <title>Running the &kea; DHCP server using &podman;</title>
+    <abstract>
+      <para>
+        &kea; is an open-source DHCP server that supports both DHCPv4 and
+        DHCPv6 protocols. It provides IPv6 prefix delegation, host reservations
+        optionally stored in a database, PXE boot, high-availability setup and
+        other features.
```
Comment (dariavladykina) [inline tasks/kea-dhcp-running-with-podman.xml:25]:
```suggestion
        DHCPv6 protocols. It provides IPv6 prefix delegation, PXE boot, 
        high-availability setup and other features, and optionally stores 
        host reservations in a database.
```

File: tasks/kea-dhcp-running-with-podman.xml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      <step>
+        <para>
+          Run the &kea; DHCP server. You can run it either using &systemd; unit
+          files, or manually.
+        </para>
+        <substeps>
+          <step>
+            <para>
+              To run &kea; as a &systemd; service, use one the following
+              commands:
```
Comment (dariavladykina) [inline tasks/kea-dhcp-running-with-podman.xml:86]:
```suggestion
              To run &kea; as a &systemd; service, use one of the following
              commands:
```

File: tasks/kea-dhcp-running-with-podman.xml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  <section xml:id="kea-dhcp-keactrl">
+    <title>The <command>keactrl</command> wrapper</title>
+    <para>
+      The installed <command>keactrl</command> wrapper uses the original
+      <command>keactrl</command> tool to send commands to deployed containers.
+      It uses the same options as the original tool but one exception: the
+      <option>-s</option> option is adjusted to send commands to the DHCPv4
+      (<option>-s dhcp4</option>) or DHCPv6 (<option>-s dhcp6</option>). If
+      <option>-s</option> is not specified, <command>keactrl</command> sends
+      commands to both servers if they are started.
```
Comment (dariavladykina) [inline tasks/kea-dhcp-running-with-podman.xml:154]:
```suggestion
      It uses the same options as the original tool with one exception: the
      <option>-s</option> option is adjusted to send commands to the DHCPv4
      (<option>-s dhcp4</option>) or DHCPv6 (<option>-s dhcp6</option>). If
      <option>-s</option> is not specified, <command>keactrl</command> sends
      commands to both servers if they are started.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Pull request merged**